tree_test.go

```diff
@@ -1178,6 +1178,7 @@ func TestEmptyHashCodeCachedPoint(t *testing.T) {
 }
 
 func TestBatchMigratedKeyValues(t *testing.T) {
+	t.SkipNow()
```

Comment:
TODO, remove this line. (Actually, I want to remove this slow test after we merge https://github.com/gballet/go-verkle/pull/349)